src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -146,8 +138,13 @@
 
     <!-- Replace the {XunitTraitOptions} place holder with the actual traits.  We use the place holder
          because code coverage needs to have a bit of the test command line after the traits (it adds ending quotes
-         to one of its options).  Simply appending the traits would break code coverage. -->
+         to one of its options).  Simply appending the traits would break code coverage. 
+         Additionally, replace CoreRun.exe with ./corerun on Non-Windows OSes (this is the only difference in the command)
+         Future refactoring will allow us to construct this correctly initially, but we don't always know the TargetOS 
+         when the properties are set currently.
+         -->
     <PropertyGroup>
+      <TestCommandLine Condition="'$(TargetOS)'!='Windows_NT'" >$(TestCommandLine.Replace('CoreRun.exe', './corerun'))</TestCommandLine>
```

Comment:
Instead of using the hardcoded string 'CoreRun.exe' here I would use the property $(TestHostExecutable).